shell/platform/windows/windows_registry.cc

```diff
@@ -0,0 +1,23 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/shell/platform/windows/windows_registry.h"
+
+namespace flutter {
+
+WindowsRegistry::WindowsRegistry() {}
+
+LSTATUS WindowsRegistry::GetRegistryValue(HKEY hkey,
+                                          LPCWSTR key,
+                                          LPCWSTR value,
+                                          DWORD flags,
+                                          LPDWORD type,
+                                          PVOID data,
+                                          LPDWORD dataSize) const {
+  return RegGetValue(hkey, key, value, flags, type, data, dataSize);
+}
+
+WindowsRegistry::~WindowsRegistry() {}
```

Comment:
Prefer default constructors/destructors if possible. See: https://stackoverflow.com/questions/21164641/difference-between-default-and-empty-constructor-with-no-arguments

```suggestion
WindowsRegistry::~WindowsRegistry() = default;
```